model/database/registry.yaml

```diff
@@ -122,6 +122,19 @@ groups:
           For batch operations, if the individual operations are known to have the same
           stored procedure name then that stored procedure name SHOULD be used.
         examples: ["GetCustomer"]
+      - id: db.operation.parameter
+        type: template[string]
+        stability: development
```

Comment:
The only place I kept `db.operation.parameter` for now is in elasticsearch semconv, so marked it as `development` for now, though could easily be convinced to keep it rc.